src/PlusDataCollection/OpenHaptics/vtkPlusOpenHapticsDevice.h

```diff
@@ -0,0 +1,104 @@
+/*=Plus=header=begin======================================================
+  Program: Plus
+  Copyright (c) Laboratory for Percutaneous Surgery. All rights reserved.
+  See License.txt for details.
+=========================================================Plus=header=end*/
+
+#ifndef __vtkPlusOpenHapticsDevice_h
+#define __vtkPlusOpenHapticsDevice_h
+
+#include "vtkPlusDataCollectionExport.h"
+
+#include "vtkPlusDevice.h"
+#include <HD/hd.h>
+#include "vtkTransform.h"
+
+class vtkMatrix4x4;
+
+struct HD_state

```

Comment:
If this is only used by this device, consider moving this into the device class to reduce global scope types.